### DISCUSSION TITLE
Migrate from CC2538 + CC2592 to CC2652

### DISCUSSION THREAD

**Discussion by X4LD1M0** [Koenkk/zigbee2mqtt#10332] [General]
I was wondering if somebody can clarify this for me. I am thinking of migrate from CC2538 + CC2592 to CC2652 (most likely the zzh!) since I am not sure it is working properly. I keep losing end-devices (Hue motion sensors) and most recently a different brand motion sensor left the network suddenly.

Would migrating to CC2652 require me to repair all my devices? My current coordinator is running 2.2.7-20201010 Z-Stack_3.0.x.

**Comment by Koenkk** [comment]
It does not require repairing, see https://www.zigbee2mqtt.io/guide/faq/#requires-repairing

**Reply by X4LD1M0** [reply]
Thanks, I did read it but after several times reading it it wasn't completely clear. That is why I raised this question here. Thanks for your quick response. I love it ever since I found it and keep tinkering with it improve my network.

Love Z2M, just couldn't resist to donate some. Merry Christmas
